packages/SwingSet/src/storageAPI.js

```diff
@@ -1,3 +1,15 @@
+/**
+ * Assert function to ensure that something expected to be a storage object
+ * actually implements the storage API.  It should have methods has, getKeys,
```

Comment:
with 'has' in first position, this reads like a typo. If the method names aren't to be quoted, it would be nice if 'has' could be later in the list.  :-)

Reply:
fixed